job_templates/ci_job.xml.em

```diff
@@ -242,10 +242,15 @@ rmdir /S /Q ws workspace "work space"
 echo "# BEGIN SECTION: Build DockerFile"
 set CONTAINER_NAME=ros2_windows_ci_%CI_ROS_DISTRO%
 set DOCKERFILE=windows_docker_resources\Dockerfile
+set SOLOFILE=windows_docker_resources\install_ros2_%CI_ROS_DISTRO%.json
+set VISUAL_STUDIO_VERSION=%CI_VISUAL_STUDIO_VERSION%
 
 rem "Change dockerfile once per day to invalidate docker caches"
 powershell "(Get-Content ${Env:DOCKERFILE}).replace('@@todays_date', $(Get-Date).ToLongDateString()) | Set-Content ${Env:DOCKERFILE}"
 
+rem "Change the solo install JSON file to contain the Visual Studio version we want to use
+powershell -noexit "(Get-Content ${Env:SOLOFILE}).replace('@vs_version', ${Env:VISUAL_STUDIO_VERSION}) | Set-Content ${Env:SOLOFILE}"
```

Comment:
Companion to an earlier suggestion.

```suggestion
powershell -noexit "(Get-Content ${Env:SOLO_FILE}).replace('@vs_version', ${Env:VISUAL_STUDIO_VERSION}) | Set-Content ${Env:SOLO_FILE}"
```

Reply:
Fixed in [b7ae4a5](https://github.com/ros2/ci/pull/786/commits/b7ae4a59d079670a8f7a8772d5ba0b3102013732)